.github/workflows/test.yml

```
@@ -0,0 +1,24 @@
+name: "Test"
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  tests:
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest]
+    runs-on: {{ '${{ matrix.os }}' }}
```

actionlint (1.7.11)

[error] expected scalar node for string value but found mapping node with "!!map" tag

(syntax-check)